README.md

```diff
@@ -134,8 +135,21 @@ require("typescript-tools").setup {
   },
 }
 ```
+Note that `handlers` can be used to override certain LSP methods. For example, you can use the `filter_diagnostics` helper to ignore specific errors:
 
-You can pass custom configuration options that will be passed to `tsserver`
+```lua
+local api = require("typescript-tools.api")
+require("typescript-tools").setup {
+  handlers = {
+    ["textDocument/publishDiagnostics"] = api.filter_diagnostics({
+      -- Ignore 'This may be converted to an async function' diagnostics.
+      { code = 80006 },
+    }),
```

Comment:
It shouldn't be like that? It is also like that in comment doc.
```suggestion
    ["textDocument/publishDiagnostics"] = api.filter_diagnostics(
      -- Ignore 'This may be converted to an async function' diagnostics.
      { 80006 }
    ),
```

Reply:
Oops you're right! I blame Copilot for that 😅